configure.ac

```diff
@@ -75,6 +75,12 @@ AC_ARG_WITH(stage1-default-location,
             [RKT_STAGE1_DEFAULT_LOCATION="${withval}"],
             [RKT_STAGE1_DEFAULT_LOCATION=auto])
 
+AC_ARG_WITH(stage1-default-images-directory,
+            [AS_HELP_STRING([--with-stage1-default-images-directory],
+                            [default directory for stage1 images, default: LIBDIR/rkt/stage1-images])],
```

Comment:
Fedora ([rkt.spec](https://github.com/fedora-cloud/rkt-rpm/blob/master/rkt.spec#L208)) at the moment has stage1 images in /usr/libexec/rkt/. This looks like a better dir than /usr/lib/rkt/stage1-images, since /usr/lib are more for .so files.